README.md

```diff
@@ -22,3 +23,53 @@ Where
 - If omitted, the link will apply to all products in the directory.
 - Test path is relative to the current directory, so will typically be just a filename.
 - Test result status is a status as defined in the wpt.fyi codebase.
+
+## How to contribute to wpt-metadata repository
+If you are a memeber of the web-platform-tests org, you can contribute to the wpt-metadata repo by sending out a PR directly or through a UI-based approach on wpt.fyi. We also have a Triage Metadata API available for trusted third parties
+
+### Triage through wpt.fyi UI
+For information about how to use the triage service on wpt.fyi, see here - UPCOMING
+
+### Triage Metadata API for trusted third parties
+In order to use the Triage Metadata API, first you need to log in on [wpt.fyi](https://wpt.fyi/) through the GitHub OAuth integration. For more information on wpt.fyi login, see [here](https://docs.google.com/document/d/1iRkaK6cGgXp3DKbNbPMVsYGMaOHO-5CfqEuLPUR_2HM)
```

Comment:
```suggestion
To use the Triage Metadata API, you first need to sign in to [wpt.fyi](https://wpt.fyi/) (top-right corner; 'Sign in with GitHub'). For more information on wpt.fyi login, see [here](https://docs.google.com/document/d/1iRkaK6cGgXp3DKbNbPMVsYGMaOHO-5CfqEuLPUR_2HM).
```

(Note also that these instructions will be wrong until prod ships sign-in. When is that happening?)

Reply:
The sign-in has shipped in the last release, although the next release is needed for api/metadata/triage to work end-to-end

Reply:
I do not see a sign-in button on https://wpt.fyi, only https://staging.wpt.fyi.

Looking at https://wpt.fyi/flags, it looks like the feature may be behind a flag. Until it is available for everyone by default, it has not shipped.

Reply:
Technically it is available by default as long as a user enables that flag themselves? since https://wpt.fyi/flags is user-side flags?

Reply:
Right, but how does a user know to go find that (hidden) page and enable that flag?

Reply:
I see. I won't submit this document until login is fully working in prod, which is most likely the next release